### PR TITLE
Update package metadata (yaml) files which still use old clone_url at…

### PR DESCRIPTION
**Pull request by alanoe**
…tribute

### CODE REVIEW

File: golang-github-russross-blackfriday/golang-github-russross-blackfriday.yaml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 Package:
- clone_url: 'https://github.com/russross/blackfriday'
- commit_id: '5f33e7b7878355cd2b7e6b8eefc48a5472c69f70'
- branch: 'master'
+ sources:
+  - git:
+     url: 'https://github.com/russross/blackfriday'
```
Comment (fabianorosas) [inline golang-github-russross-blackfriday/golang-github-russross-blackfriday.yaml]: src

File: golang-github-shurcooL-sanitized_anchor_name/golang-github-shurcooL-sanitized_anchor_name.yaml [hunk 2]
```diff
@@ -1,7 +1,9 @@
 Package:
- clone_url: 'https://github.com/shurcooL/sanitized_anchor_name'
- commit_id: '1dba4b3954bc059efc3991ec364f9f9a35f597d2'
- branch: 'master'
+ sources:
+  - git:
+     url: 'https://github.com/shurcooL/sanitized_anchor_name'
```
Comment (fabianorosas) [inline golang-github-shurcooL-sanitized_anchor_name/golang-github-shurcooL-sanitized_anchor_name.yaml]: src

File: golang-github-russross-blackfriday/golang-github-russross-blackfriday.yaml (hunk 1 above)
Comment (alanoe) [inline golang-github-russross-blackfriday/golang-github-russross-blackfriday.yaml]: Done

File: golang-github-shurcooL-sanitized_anchor_name/golang-github-shurcooL-sanitized_anchor_name.yaml (hunk 2 above)
Comment (alanoe) [inline golang-github-shurcooL-sanitized_anchor_name/golang-github-shurcooL-sanitized_anchor_name.yaml]: Done

**Pull request merged**